app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java

```diff
@@ -35,8 +35,11 @@ public CircumventionSuite(){
                 list.add(new Psiphon());
             if (pm == null || pm.isTestTor())
                 list.add(new Tor());
-            if (pm == null || pm.isTestRiseupVPN())
-                list.add(new RiseupVPN());
+            /* TODO (aanorbel): Riseup VPN Disabled.
+                The riseupvpn experiment has been quite flaky for quite some time.
+                To be enabled only when test is fixed or removed if deemed necessary.
+                if (pm == null || pm.isTestRiseupVPN())
+                list.add(new RiseupVPN());*/
```

Comment:
I'd just remove the commented out code: we can always go back and see the git history to see how we removed riseupvpn to learn how to add it back.